src/component/Header.svelte

```diff
@@ -0,0 +1,123 @@
+<script>  
+    let srcHeader = '././img/image-header.jpg'; 
+    let srcArrow = '././img/icon-arrow-down.svg';
+    let srcLogo = '././img/logo.svg';
+</script>
+
+<header>
+    <img src={srcHeader} class='srcHeader' alt='HeaderPhoto'>
+    <h1>WE ARE CREATIVES</h1>
+    <img src={srcArrow} class='arrow' alt='ArrowPhoto'>
+    <div class='header-top'>
+        <img src={srcLogo} class='logo' alt='SunnysideLogo'>
+        <navbar>
+            <ul>
+                <li><a class='about' href="#about">About</a></li>
+                <li><a class='services' href="#services">Services</a></li>
+                <li><a class='projects' href="#projects">Projects</a></li>
+                <button class='contact'>CONTACT </button>
+            </ul>
+        </navbar>
+    </div>         
+</header>
+
+<style>
+    header {
+        position: relative;
+        width: 1440px;
+        height: 800px;
+        left: 0px;
+        top: 0px;
+    }
+    .srcHeader {
+        width: 1440px;
+        height: 800px;
+        left: 0px;
+        top: 0px;
+        z-index:-1;
```

Comment:
Bunu niye ekledin?

Reply:
Header'ın background image'ını konumlandırmak ve ölçülendirmek için height, left ve top kullandım. Yazıların altında kalması için de z-index kullandım.